Library/Homebrew/os/mac/xcode.rb

```diff
@@ -270,8 +270,10 @@ def update_instructions
             sudo rm -rf /Library/Developer/CommandLineTools
             sudo xcode-select --install
 
-          Alternatively, manually download them from:
+          Alternatively (or if other options fail), manually download them from:
             #{Formatter.url("https://developer.apple.com/download/more/")}.
+          And refer to the following for the most recent available for your OS:
+            #{Formatter.url("https://en.wikipedia.org/wiki/Xcode#Version_comparison_table")}.
```

Comment:
Did you personally find this graph helpful? I wasn't sure what to do based on it.

Reply:
It's confusing unless you already know what the correct answer is and the graph confirms it after you stare at it for a few minutes :).

Reply:
I think it may be worth omitting based on that. Perhaps we can just output the minimum version that we were expecting?

Reply:
(see `outdated?` for the comparison that's used to determine this)